android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -257,56 +259,80 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
       }
     });
 
-    return webView;
+    return wrapper;
   }
 
   @ReactProp(name = "javaScriptEnabled")
-  public void setJavaScriptEnabled(WebView view, boolean enabled) {
-    view.getSettings().setJavaScriptEnabled(enabled);
+  public void setJavaScriptEnabled(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setJavaScriptEnabled(enabled);
   }
 
   @ReactProp(name = "setBuiltInZoomControls")
-  public void setBuiltInZoomControls(WebView view, boolean enabled) {
-    view.getSettings().setBuiltInZoomControls(enabled);
+  public void setBuiltInZoomControls(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setBuiltInZoomControls(enabled);
   }
 
   @ReactProp(name = "setDisplayZoomControls")
-  public void setDisplayZoomControls(WebView view, boolean enabled) {
-    view.getSettings().setDisplayZoomControls(enabled);
+  public void setDisplayZoomControls(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setDisplayZoomControls(enabled);
   }
 
   @ReactProp(name = "setSupportMultipleWindows")
-  public void setSupportMultipleWindows(WebView view, boolean enabled){
-    view.getSettings().setSupportMultipleWindows(enabled);
+  public void setSupportMultipleWindows(RNCWebView view, boolean enabled){
```

Comment:
I think that case would only realistically happen if the webview was released but still mounted. I can't think of a use case where that would be required. I prefer if we don't add this functionality (since we have some logging)

Reply:
> I prefer if we don't add this functionality (since we have some logging)

That works for me. As long as we can detect when our assumptions are wrong through the logging, I'm cool with assuming that `view.getWebView()` will be non-null here